### ISSUE TITLE
Platform priority not taken into account when newer version is available for one platform

### ISSUE THREAD

**Issue by murasame78** [Sude-/lgogdownloader#88]
I regularly use the following command :

`lgogdownloader --platform linux,windows --language fr,en --download --game all` 

It does the job very well by downloading linux version of my games when available and otherwise downloading the windows version.

Yet, I recently noticed that the windows version would be downloaded regardless of the priority when it has a  more recent version number (e.g: Trine Enchanted Edition is marked as 2.1.0.5 for windows and 2.0.0.2 for linux).

As a result, both version are downloaded. Not sure it is the intended behaviour.

**Comment by Sude-** [comment]
This is not intended behaviour.
I haven't been able to replicate this and gameDetails::filterListWithPriorities doesn't have any code for handling version numbers or updates so version number shouldn't be a factor.

Because platform and language priorities are scored in the same pass I'm guessing that Windows installer has French language but Linux installer doesn't and therefore both are getting the same score in gameDetails::filterListWithPriorities

```--platform linux,windows --language fr,en``` would give scores
Linux = 0
Windows = 1
French = 0
English = 1

Linux installer score: Linux + English = 0 + 1 = 1
Windows installer score: Windows + French = 1 + 0 = 1
They have the same score so both are downloaded

Could you try if changing the scoring system helps and doesn't cause any other issues?
Change 
```fileDetails->score += i;```
on [line 48 of src/gamedetails.cpp](https://github.com/Sude-/lgogdownloader/blob/0b455dd/src/gamedetails.cpp#L48) to
```fileDetails->score += i*2;```

This change makes platform affect the score more and thus the new scores for ```--platform linux,windows --language fr,en``` are
Linux = 0
Windows = 2
French = 0
English = 1

Linux installer score: Linux + English = 0 + 1 = 1
Windows installer score: Windows + French = 2 + 0 = 2
Score for Linux installer is less than Windows installer so only the Linux installer should be downloaded.

**Comment by murasame78** [comment]
Indeed, that did the trick ! I've just updated my whole game collection (including "Trine") and the behaviour was as expected. It only downloaded windows version where no linux version was available and took into account my language preference.

Thanks !

Now will you include this change in your next release ?
